packages/web/src/components/spa/HackAUBG/VerifyAccountPop/VerifyAccount.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import styles from './verify_account.module.css';
+import { FaRegWindowMinimize } from 'react-icons/fa';
+import { useLocation } from 'react-router-dom';
+import axios from 'axios';
+import { verifyURL } from '../../../../Global';
+
+export const VerifyAccount = ({ onSuccess }) => {
+    const location = useLocation();
+    const params = new URLSearchParams(location.search);
+    const jwtToken = params.get('jwt_token');
```

Comment:
Never mind my previous comment, I see that you're already doing it.
What happens if the jwt token is not present? I suggest that the modal shows a message saying, the security token was not provisioned, email can't be verified or osmething of the sorts